From: Sedat Dilek <sedat.dilek@gmail.com>

With OverlayFS v22 I get the following build error when
CONFIG_OVERLAYFS_FS=m

ERROR: "d_ancestor" [fs/overlayfs/overlayfs.ko] undefined!

Fix this by adding the missing export.

Signed-off-by: Miklos Szeredi <mszeredi@suse.cz>
---
 fs/dcache.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fs/dcache.c b/fs/dcache.c
index 42ae01eefc07..b233bfc849de 100644
--- a/fs/dcache.c
+++ b/fs/dcache.c
@@ -2559,6 +2559,7 @@ struct dentry *d_ancestor(struct dentry *p1, struct dentry *p2)
 	}
 	return NULL;
 }
+EXPORT_SYMBOL(d_ancestor);
 
 /*
  * This helper attempts to cope with remotely renamed directories
-- 
1.8.1.4
